homeassistant/components/remote_rpi_gpio/binary_sensor.py

```diff
@@ -4,6 +4,7 @@
 import requests
 import voluptuous as vol
 
+from homeassistant.components import remote_rpi_gpio
```

Comment:
Shouldn't we import the names we need relatively from the package module instead? The most important point with the relative intra package import is that we can override the integration as a custom integration and it will still work.

Reply:
I have adjusted the plugin, and added some tests.
Should be ok now.